README.md

```diff
@@ -3,5 +3,29 @@ Repositorio para la asignatura Infraestructura Virtual de Antonio Rodríguez Rod
 
 ## Problema a resolver:
 
-Soy un estudiante que tiene el problema de ser muy exquisito con la comida y quiero saber cuando se va a ofrecer comida que se adapte a mis preferencias, ya que pierdo mucho tiempo teniendo que entrar en la web de comedores universitarios y ver entre todo lo disponible si me gusta algo. 
-Del mismo modo, me gustaría saber la calidad alimenticia de los platos que hay disponibles cada día: macronutrientes, calorías, etc. Ya que tengo como objetivo personal, con mi entrenamiento, subir de peso ganando masa muscular, con lo que debería tener alguna forma de adecuar mi dieta a dicho objetivo.
\ No newline at end of file
+Soy un estudiante que tiene el problema de ser muy exquisito con la comida y quiero saber cuando se va a ofrecer comida que se adapte a mis preferencias, ya que pierdo mucho tiempo teniendo que entrar en la web de comedores universitarios y ver entre todo lo disponible si me gusta algo. Del mismo modo, me gustaría saber la calidad alimenticia de los platos que hay disponibles cada día: macronutrientes, calorías, etc. Ya que tengo como objetivo personal, con mi entrenamiento, subir de peso ganando masa muscular, con lo que debería tener alguna forma de adecuar mi dieta a dicho objetivo.
+
+# Resolución del problema
+
+## Descripción
+Este proyecto es un sistema automatizado que ayuda a los estudiantes a encontrar comidas que se adapten a sus preferencias y objetivos nutricionales. El sistema extrae los menús diarios de diferentes comedores disponibles, filtra los platos según las preferencias del usuario y proporciona un análisis nutricional de los platos.
+El estudiante evita perder tiempo revisando los menús manualmente y puede seguir un plan nutricional adecuado a sus necesidades de entrenamiento.
+
+## [Journey](https://github.com/antoniorr02/MenuConsulter/blob/Objetivo-1/docs/journeys.md)
+- Un estudiante universitario, es muy selectivo con la comida y quiere asegurarse de que cada día haya opciones que le gusten en el comedor. Tiene poco tiempo y no le gusta revisar los menús manualmente.
+- Una estudiante que entrena para ganar masa muscular, quiere asegurarse de que las comidas que toma contribuyen a su objetivo de consumo de proteínas y calorías.
+
+## [Historias de Usuario](https://github.com/antoniorr02/MenuConsulter/blob/Objetivo-1/docs/historias_usuario.md)
+- **[HU1]:** Ojalá pudiera ver rápidamente todos los menús de los diferentes comedores que tenga marcados como favoritos.
+- **[HU2]:** Ojalá me avisaran cuando haya platos que se ajusten a mis preferencias alimenticias.
```

Comment:
¿Estás seguro de que puedes casar "preferencias alimenticias" con los nombres de los platos? ¿Qué datos adicionales necesitarías para eso?
para estas cosas, tanto en el user journey como en la HU, la metodología personas te dice que tienes que hablar de una persona concreta y de preferencias concretas. Si es que es vegetariana, por ejemplo, hay menús específicos para vegetarianos. Pero si es que le gusta la tortilla de patatas, la historia es totalmente diferente.
Por favor, mira la metodología personas para poder expresar mejor estas historias de usuario (teniendo en cuenta que es muy probable que sólo te dé tiempo a una sola HU durante la asignatura)

Reply:
Se me ocurren diferente formas de hacerlo, como por ejemplo utilizar cada plato de cada menú como un objeto seleccionable por el usuario para marcarlo como favorito. En cualquier caso esto quedaría en el ámbito de un producto más maduro y una fase del desarrollo de la aplicación posterior. Con ello, voy a seguir su consejo y voy a orientar los journeys y las HU a usuarios objetivos concretos.

Reply:
Una vez más, tienes que hacer cambios en la línea. El "ojalá" es una "rule of thumb" que no tiene por qué ponerse explícitamente en la HU, por cierto.